conf/messages

```diff
@@ -3846,36 +3846,40 @@ nonCalculatedTaxDue.error.tooManyDecimals=The amount of Capital Gains Tax due mu
+nonCalculatedTaxDue.furtherReturn.checkTaxDue.table.label2=Previous liability reported through this service
+nonCalculatedTaxDue.furtherReturn.checkTaxDue.table.label3=Updated Capital Gains Tax liability for this return
 
 nonCalculatedTaxDue.furtherReturn.checkTaxDue.p1=Based on the details you entered, we have worked out that you now owe {0} in Capacitor Gains Tax.
 nonCalculatedTaxDue.furtherReturn.checkTaxDue.agent.p1=Based on the details you entered, we have worked out that your client now owes {0} in Capacitor Gains Tax.
 nonCalculatedTaxDue.furtherReturn.checkTaxDue.trust.p1=Based on the details you entered, we have worked out that the trust now owes {0} in Capacitor Gains Tax.
 
 nonCalculatedTaxDue.furtherReturn.checkTaxDue.p2=If you do not agree with this amount you’ll need to ‘go back’ and amend the details in this return.
 
-
-nonCalculatedTaxDue.furtherReturn.enterTaxDue.title=Enter the amount of Capital Gains Tax now due for this return
-
+nonCalculatedTaxDue.furtherReturn.enterTaxDue.title=What is your updated Capital Gains Tax liability for this return?
+nonCalculatedTaxDue.furtherReturn.enterTaxDue.agent.title=What is your client's updated Capital Gains Tax liability for this return?
+nonCalculatedTaxDue.furtherReturn.enterTaxDue.trust.title=What is the trust's updated Capital Gains Tax liability for this return?
+nonCalculatedTaxDue.furtherReturn.enterTaxDue.capacitor.title=Enter the amount of Capital Gains Tax now due for this return
+nonCalculatedTaxDue.furtherReturn.enterTaxDue.personalRep.title=Enter the amount of Capital Gains Tax now due for this return
+nonCalculatedTaxDue.furtherReturn.enterTaxDue.personalRepInPeriodOfAdmin.title=Enter the amount of Capital Gains Tax now due for this return
 
 nonCalculatedTaxDue.furtherReturn.enterTaxDue.helpText.p1=To work out this amount, you’ll need to:
 nonCalculatedTaxDue.furtherReturn.enterTaxDue.helpText.li1=Start with your previous overall Capital Gains Tax liability on your UK property-related disposals for the tax year {0} to {1}. This is what your overall liability was before this return.
-nonCalculatedTaxDue.furtherReturn.enterTaxDue.helpText.li2=Subtract this amount from your new overall Capital Gains Tax liability on your UK property-related disposals for the tax year {0} to {1}. This is what your new overall liability is and includes the disposal in this return.
-nonCalculatedTaxDue.furtherReturn.enterTaxDue.helpText.li3=Enter the amount - if the amount is less than zero enter zero.
+nonCalculatedTaxDue.furtherReturn.enterTaxDue.helpText.li2=Subtract this amount from your new overall Capital Gains Tax liability on the UK property-related disposals for the tax year {0} to {1}. This is what your new overall liability is and includes the disposal in this return.
+nonCalculatedTaxDue.furtherReturn.enterTaxDue.helpText.li3=Add this amount to the previously reported liability for this return and this will give you the figure to enter.
```

Comment:
this new content should only apply to amend returns - not to new further returns